lib/Controller/SettingsController.php

```diff
@@ -17,6 +17,7 @@
 use OCP\AppFramework\Controller;
 use OCP\AppFramework\Db\DoesNotExistException;
 use OCP\AppFramework\Http;
+use OCP\AppFramework\Http\Attribute\PasswordConfirmationRequired;
```

Comment:
This line should be deleted :wink:.
After having committed this change, you can squash the 3 last commits together with `git rebase -i HEAD~3` and force push.